backend/.gitignore

```diff
@@ -1,2 +1 @@
-.env
-/src/schema.rs
\ No newline at end of file
+/src/schema.rs
```

Comment:
Why do we ignore this file.
I know this is generated by `diesel` but why keep it out of VCS?

Reply:
Since its regenerated more or less every time a diesel cli command is executed, there is no point in keeping track of it in git.

AFAIK (I'm certainly not an expert, so please feel free to correct me dear rust experts), schema.rs is supposed to mirror the current state of your database schema with rust structs, hence the name.
All of the information contained within it is already stored in  the "migrations" folder, just in the form of SQL scripts.

I hope that answers your question.